### DISCUSSION TITLE
Can the vscode extension be open sourced?

### DISCUSSION THREAD

**Discussion by smehta2000** [microsoft/promptflow#3392] [Ideas]
We would like to add custom features to the extension, such as new node types, more A/B testing features etc.

**Comment by JYC-99** [comment]
There isn't any opensource plan for the VS Code extension or Web UI supports yet.
However, we highly value feedback and feature requests. Please provide a detailed description of the features you would like to add. If your suggestion aligns with our roadmap and receives approval, it will initiate our development process. Thank you for your interest and contribution to improving our services.
